client/common/components/BoardSection.tsx

```diff
@@ -27,31 +31,68 @@ const BoardSection: React.FC<BoardInterface> = (props) => {
     setCurrentTask(task);
   };
 
-  const onClick = () => {
-    console.log('hi');
+  const handleShowInput = () => {
+    setShowInput(!showInput);
+    setShowButton(false);
   };
+
+  const handleCreateTask = async () => {
+    await createNewTask(taskTitle);
```

Comment:
should we also pass in the other inputs?

Reply:
yess :') trying for now